tracing-subscriber/src/filter/targets.rs

```diff
@@ -1,3 +1,10 @@
+//! A filter that enables or disables spans and events based on their [target] and [level].
+//!
+//! See [`Targets`] for details.
+//!
+//! [target]: tracing_core::Metadata::target
+//! [level]: tracing_core::Level
```

Comment:
Nit: maybe this should have a link to the `Filter` docs? But, the "see `Targets` for details" link is probably sufficient...
```suggestion
//! A [filter] that enables or disables spans and events based on their [target] and [level].
//!
//! See [`Targets`] for details.
//!
//! [target]: tracing_core::Metadata::target
//! [level]: tracing_core::Level
//! [filter]: crate::layer#filtering-with-layers
```